app/models/assign_sp_component_to_service_event.rb

```diff
@@ -0,0 +1,19 @@
+class AssignSpComponentToServiceEvent < AggregatedEvent
+  belongs_to_aggregate :service
+  data_attributes :sp_component_id
+  validate :component_is_correct_type?
+
+  def attributes_to_apply
+    {
+      sp_component_id: sp_component_id,
```

Comment:
would it be useful to also have the service name in here? I know we could find it through querying the `sp_component_id`, but could be useful to be able to see the service name at first glance in the events view.

Reply:
you should be able to do it just adding, the name in the `data_attributes` on line 3...